crates/solidity/testing/snapshots/cst_output/Block/unchecked/generated/0.6.2-failure.yml

```diff
@@ -0,0 +1,41 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ { unchecked { x = 1; } }                                                         │ 0..24
+
+Errors: # 2 total
+  - >
+    Error: Expected Colon.
+       ╭─[crates/solidity/testing/snapshots/cst_output/Block/unchecked/input.sol:1:17]
+       │
+     1 │ { unchecked { x = 1; } }
+       │                 ──┬──  
+       │                   ╰──── Error occurred here.
+    ───╯
+  - >
+    Error: Expected OpenBrace or OpenParen.
```

Comment:
not sure why are we expecting `OpenBrace` or `OpenParen` here. Since function call expressions are postfix operators, shouldn't we be skipping it alltogether? am I missing something?